plugins/wporg-5ftf/includes/email.php

```diff
@@ -78,10 +95,14 @@ function send_email( $to, $subject, $message, $pledge_id ) {
  * @param string $action
  * @param int    $action_page_id The ID of the page that the user will be taken back to, in order to process their
  *                               confirmation request.
+ * @param bool   $use_once       Whether or not the token should be deleted after the first use. Only pass `false`
+ *                               when the action requires several steps in a flow, rather than a single step. For
+ *                               instance, be able to 1) view a private pledge; 2) make changes and save them; and
+ *                               3) reload the private pledge with the new changes displayed.
  *
  * @return string
  */
-function get_authentication_url( $pledge_id, $action, $action_page_id ) {
+function get_authentication_url( $pledge_id, $action, $action_page_id, $use_once = true ) {
```

Comment:
Do you have a scenario in mind where this new flag would need to be set to false? Might be best to leave it off until the need arises to prevent unnecessary code paths.

Reply:
Yeah, it'll be used when emailing pledge admins a link to update their pledge. It needs to be reusable then so that they can go through the `open pledge -> submit updated pledge -> view updated pledge` flow.

https://github.com/WordPress/five-for-the-future/pull/75/commits/d4418861009988a8dd73f5c61f4bdb7512de426e#diff-89dbb624f06d56ec75b35419157c3842R280